src/status_im2/navigation/screens.cljs

```diff
@@ -29,7 +30,12 @@
 
            {:name      :community-overview
             :options   {:topBar {:visible false}}
-            :component communities.overview/overview}]
+            :component communities.overview/overview}
+
+           {:name      :settings-syncing
+            :insets    {:bottom true}
+            :options   {:topBar {:title {:text (i18n/label :syncing)}}}
```

Comment:
wow nice catch !
This never caught my attention, probably because it did not result in either a compile time or a runtime error.

fixed this in latest commit